### ISSUE TITLE
Check site for accessibility issues

### ISSUE THREAD

**Issue by tbl73** [benwbrum/fromthepage#671]
Run http://wave.webaim.org report and fix the accessibility issues.

**Comment by saracarl** [comment]
waiting on funding.

**Comment by tbl73** [comment]
I've gotten through what I could, but there are some errors left:

Not fixable (without bigger changes):
- There are contrast errors throughout the site, which come from text colors not being different enough.  Many of them are due to the "faded" font that we use for informational purposes.  
- The overview of particular pages come up with an error that says "missing or uninformative page title".  That's true, but it's because many pages do have uninformative titles unless they are renamed.  (Untitled Page 1 isn't terribly informative, but it is what the page is called.)

Things I haven't been able to figure out:
- Collection - Settings - Image upload:  If there isn't an image, it says there's an empty form label, but it's fine if there is an image.  I added an aria-label, but that didn't take care of it.  There is a label there, but putting a title in seems to make no difference.  It's sort of a weird set up with the CSS and data-attributes, though, and I can't work out how to change it so that the title will show up.
- Work - settings - Image upload - same problem
- Article Overview - "image map area needs alt text" - the clickable links to the other articles are map areas which should have alt-text.  It's got to go in the graph.dot file (I think), but I haven't been able to find a way to put it in there..
- In IIIF imported works, the display page (in overview/transcribe/translate) - the zoom icons apparently are in some kind of form, and it doesn't have a title.  That's built by Openseadragon, though, and I couldn't find a place to fix it.  (The code for openseadragon is like a giant run on sentence, which doesn't help.)

Changes to note:
- The multiselect select2 dropdown boxes that we use for categorizing articles - I ended up having to change the code in the select2 JS plugin to get rid of the error (another no form label or empty form label error).
- I had to add a line to the bookreader plugin code to give IA imported page images alt text.
